Dockerfile

```diff
@@ -17,6 +15,3 @@ ADD . /var/node/mkimage-server
 
 # define default command
 CMD node app.js
-
-# expose ports
-EXPOSE 80
```

Comment:
Is this going to be the convention moving forward? Should I remove EXPOSE commands from Dockerfiles as I come across them?